runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersPlugin.java

```diff
@@ -518,6 +529,46 @@ public void onGameTick(GameTick event)
 		Player player = client.getLocalPlayer();
 		WorldPoint currentWorldPoint = player.getWorldLocation();
 
+		SkullIcon currentSkullIcon = player.getSkullIcon();
+		int currentRegionID = WorldPoint.fromLocalInstance(client, client.getLocalPlayer().getLocalLocation()).getRegionID();
+		final int abyssRegionID = 12107;
+		Item[] equippedItems = client.getItemContainer(InventoryID.EQUIPMENT).getItems();
+		boolean avariceThisTick;
+
+		if (equippedItems[2].getId() == ItemID.AMULET_OF_AVARICE)
+		{
+			avariceThisTick = true;
+		}
+		else
+		{
+			avariceThisTick = false;
+		}
+
+		if (currentRegionID != lastRegionID && currentRegionID == abyssRegionID && currentSkullIcon == SkullIcon.SKULL)
```

Comment:
not sure if you want to but for this series of if/else clauses it might be a good idea to comment each one with each of the wilderness skull interactions it is checking for - that way it's easier for maintainers to know what skull check each is performing. I.e. something like for this line `else if (avariceLastTick == true && avariceThisTick == false)` could be "only start the skull timer for the amulet of avarice once it is removed since the timer won't start ticking down until the amulet is taken off"